speckle/SpeckleClient.py

```diff
@@ -33,6 +33,10 @@ def check_response_status_code(self, r):
             self.log(json.dumps(r.text, indent=4, sort_keys=True))            
         return False
 
+
+    '''
```

Comment:
That looks like a breaking change to me. Probably worth notifying it in the commit.

Reply:
I changed the PR comments to include BREAKING CHANGE. Hope I did it correctly...

Reply:
PR comment won't affect the semantic-release process unfortunately. Only the commit messages count. However we could squash and merge this to make it work. Or, even easier, you could install commitizen on your machine and create commits using it. You'll need to install nodejs and then do the following:

npm install -g commitizen cz-conventional-changelog
go to your pyspeckle folder
* make some changes
* git add .
* git cz and follow the command line prompt

Trust me, this will make your life so much easier in terms of this commit format.